modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -242,7 +248,8 @@ sudo apt install phpmyadmin
 
 === Start a Service After a Resource is Mounted
 
-If you have network resources, such as NFS or iSCSI based mounts and you want to make sure that the database or web server only starts _after_ the resource is mounted.
+If you have network resources, such as NFS or iSCSI based mounts and you want to make
+sure that thedatabase or web server only starts _after_ the resource is mounted.
```

Comment:
```suggestion
sure that the database or web server only starts _after_ the resource is mounted,
```
(this sentence was already ending with a full stop, but did not have any "then" clause)